### ISSUE TITLE
8.媒介传导——怎样联系上帝的精神世界

### ISSUE THREAD

**Issue by Pathwork-In-China** [Pathwork-In-China/Pathwork-Lectures#10]
1. Mediumship—How to Contact God’s Spirit World
### 媒介传导——怎样联系上帝的精神世界
Pathwork Guide Lecture No. 8 | July 01, 1957
心路指导讲座 第8篇 | 1957.7.1

Greetings in the name of God. I bring you divine blessings.
以上帝的之名问候你们。我为你们带来神圣的祝福。

My dear friends, last time I spoke to you about entering the first gate that leads to the path of perfection. I addressed the problem of making the decision and committing yourself to it without reservations. I explained how to overcome the difficulties. Since then I have observed you, my friends, and I can tell you that for God’s spirits it is a great joy to witness whenever the first step has been taken and the decision made. Today I would like to tell you a few words about what happens after.
我亲爱的朋友们，上次我与你们谈到了关于进入通往完善状态道路的第一道大门。我说到做出决定并无保留地把自己投入其中的问题。我解释了怎样克服其中的困难。从那之后我有在观察你们，我的朋友们，而且我可以告诉你们的是，对于上帝的魂灵们来说，每看到有人迈出第一步并且做出决定，都是一份重大的喜悦。今天我想告诉你们一些这之后发生的事情。

First, however, I would like to add that we also see when our friends who hear and read these words avoid the decision and believe it has nothing to do with them. They do not even attempt to make a commitment. This reaction imprints itself upon the soul and thus has its consequences for the individual’s fate. Then there are also some friends who believe that they have already been on this path for a long time, just because now and again they say a prayer, or because they take pains to become decent human beings. Yet this was not my meaning. The decision I was referring to means more. Therefore, I say to these friends specifically—and not only to the ones present—consider where you stand. Take into account the possibility that you might not be exactly in the place where you wish you were.
但是首先我想补充的是，我们也会看到我们的朋友们听到或读到这些话后避开做出决定，并认为这与他们没什么关系。他们甚至没有尝试做出承诺与投入。这种反应会铭刻在人的灵魂上，因而会对这个人的命运产生相应的后果。然后也有一些朋友们相信，他们已经在这条道路上进行了很长时间了，仅仅因为他们偶尔会说句祈祷的话，或者因为他们为成为体面的人而经受了痛苦。然而这不是我的意思。我提到的决定有更深的含义。因此，我现在特别对这些朋友们——而且并不是仅针对在座的——说：考虑下你所处的位置。考虑下这种可能性：你也许并不完全处在你希望处在的地方。

Those who have made the decision and given themselves into the hands of God immediately sense what is happening within. The first reaction is a great relief and an inward joy that for a while completely takes possession of one. This wonderful feeling, though, does not stay with you forever. Hearing this, you could easily conclude, at least in your feelings, that if the heaviness and unpleasantness is likely to return, there is no point in making the decision in the first place. But I say to you that this is not so. After having crossed the first threshold, you are only at the beginning of the path. I have told you before that the spiritual work only begins after this crossing, and that being on this path consists in not giving up, but continuing to press on steadily. That has to be the price.
那些已经做出决定并把自己放入上帝的手中的人，会立刻感到内心发生的变化。第一反应是一种重大的宽慰和放松，以及一种向内的喜悦，这种喜悦会在一段时间里彻底占据一个人。然而，这种美妙的感觉不会永远保持在你身上。听到这，你可能会——至少是在感觉上——轻易地得出结论，认为如果那些沉重感和不愉快感很可能会回来，那么做出这个决定一开始就没有意义。但我对你们说，不是这样的。在跨越了第一个临界点后，你仅仅处在这个道路的起点上而已。我已经告诉过你们精神工作只有在这个跨越之后才开始展开，而处于这条道路上意味着不仅不能放弃它，更要继续稳定向前迈进。这是必须付出的代价。

Today I will not speak much about what to do when you enter the path of perfection, happiness and liberation, because I have done so in the past. I will rather speak about something I have not yet said much of—the good and beautiful things you can expect on this path. You already know about the difficulties, about carrying out the resolutions and sticking with them even though at times that appears hard to do, about not losing your resolve, but to recommit both time and willpower again and again. If you do this, the beautiful reward becomes more and more tangible. This will happen very slowly, not as a sudden experience.
今天我不会就“你进入了通向完善、幸福和解放的道路时应该做什么”这个话题说太多，因为我已经说过了。现在我会说一些没说过的话题——那些在这条道路上你可以预料、期望的有益又美丽的事情。你已了解其中的困难；了解了要下定决心并坚持它们，哪怕有时候这样做似乎很难；了解了要保持你们的决意，而且要一次又一次地重新投入时间和意志力。如果你这样做的话，美妙的奖赏会变得越来越触手可及。这会逐渐、慢慢地发生，而不会是一种突然性的经历。

When the spiritual world recognizes that a human being is serious about the decision and does not stay with the good intent alone but translates it into action in spite of unavoidable relapses and occasional failures, it will then give increased spiritual protection. This, too, is spiritual law. Whoever goes on this arduous path needs a different kind of spiritual help than the person who does not. The person on the path will either be given an additional, higher, and more powerful spirit guide, or there will be an exchange of guardians. The former spirit, having fulfilled its obligation by bringing the human being over the threshold, is now ready for another task.
当精神世界意识到一个人类个体严肃认真地对待这个决定，不是仅仅停留在单纯的好意上而是把它翻译成行动，哪怕会些无法避免的故态复萌和偶尔的失败也没有关系时，精神世界就会给强化的精神保护。这也是一条精神法则。不论是谁走在这条艰巨的道路上，都会需要与不参与这条道路的人不同种类的精神帮助。处在道路上的人要么会被给予一位额外的、更高层次、更强力的精神向导，要么就会有一次向导间的替换。那位之前的魂灵，通过把这个人带入这个门槛而完成了自己的义务，现在已可以接收下个任务了。

Such an exchange may actually have taken place before passing through this particular gate. When an increased spiritual protection is given, the person will be absolutely sure of its presence. A strengthening of protection can be given several times during a lifetime, whenever important tests have to be passed or spiritual choices made, either before or after crossing the threshold. There are several crossroads in an individual’s life. Yet even if you have clearly had such an experience of strengthened spiritual protection, do not take it as proof that you have definitely crossed the threshold of the entrance to the path of perfection.
这种替换可能在穿越这个特定大门前就发生了。增强的精神保护就位后，人会毫无疑问地确定它的存在。保护的加强可能在一生中发生数次，每当要经历重要的测验或者要做出精神层面的决定时，要么是临界点之前要么之后，这种事情就会发生。一个人生命中有数个十字路口。但是，就算你已经很清楚地有过这种加强精神保护的经历，也不要把它当成是“你已经明确无误地跨过了进入自我完善道路的门槛”的确定性证明。

You must understand exactly what this path actually is. It is an intensive, continuous work on one’s own development; it is getting to know the self; it is, first and foremost, the acceptance of one’s imperfections. Then follows the inward integration of spiritual laws which with time will affect and alter the imperfect emotional currents in the soul. Who does not work in this way every day, with outer help and advice without which the work cannot progress at first, does not walk this path. I must stress this again and again for certain friends.
你必须准确无误地理解这条道路实际是什么。它是对于个人自我发展的高强度、持续性的工作；它是逐步认识自己的过程；它——首先，也是最重要的——是对于自己不完美之处的接纳。然后随之而来的是对精神法则的内部整合，这种整合随着时间的推移，会影响和更改灵魂内不完美的情绪涌流。一个人如果不是每天都以这种方式自我工作，并利用好外部帮助和建议（没有这个元素这份工作一开始不会有进展），那他就没有行走在这条道路上。我必须针对某些朋友一再强调这点。

Furthermore, the human beings who enjoy the strengthened protection will, when striving to establish the contact with God’s spirit world—which is, after all, what it’s all about—receive meaningful knowledge or intuitive insights from the spiritual realm. Such guidance will no longer come in unconnected glimpses occurring here and there along the way even before this particular threshold has been crossed, but will become a comprehensible part of a greater whole as the interconnectedness of its system and order is increasingly understood. These people will grasp evermore clearly how their spiritual path should proceed, how they should work on themselves, what to tackle first. They will perceive what the real alternatives are in their lives. They will come to see God’s will in general, as well as the meaning and the purpose of the tests for them personally.
此外，对那些享受到强化保护的人，当他们试图与上帝的精神世界建立联络时——而这归根结底就是其中的实质——就会从精神世界领域收到有意义的知识和直觉性的洞察。这种指导不再会以道路上零星分布的互不相关的窥视的形式发生了（那种方式在跨过这个特定的门槛前就能得到），而是说它会成为更大整体背景之下易于理解的一部分，因为它系统性和有序性的内部联系被愈加清楚地理解了。这些人会越加清晰地了解到他们的精神道路应该怎样行进，他们应该怎样在自己身上做工作，以及先处理哪些问题。他们会意识到生活中真正的选择余地是什么。他们会普遍性地看到上帝的意志是什么，以及他们个人考验的意义和目的。

This discovery is very important because it marks a great turning point in human life. Before this point, the tests, the small and great misfortunes of life, were incomprehensible and that made life very difficult. After passing a turning point it is as if a veil had lifted, and the person is able to overcome difficulties more easily. Moreover, the individuals will receive help with their strictly mundane problems and difficulties—something which before was not possible. I can tell you, my dear friends, that the happiness, the joy, the inner liberation for each victory gained in this struggle cannot be compared with anything else human beings can ever experience.
这种发现有着重大意义，因为它标志了人类生命中的重大转折点。在这个节点之前，那些考验，以及生活中或大或小的不幸，都是不易于理解的，进而使生活变得很艰难。在穿过一个转折点以后，就好像有一个面纱帷幕被掀开了，这个人变得能更轻易地克服困难。再者，这些个体会在他们严格意义上的世俗问题和困难中得到帮助——这种东西在转折点之前是不可能的。我可以告诉你们的是，我亲爱的朋友们，在这个奋斗中获得胜利后得到的那种幸福快乐、那种喜悦以及内在的解放，是人类能够经历的任何其他东西都无法与之相比的。

Since the goal of my teachings is to make the contact with God’s spirit world possible for each individual, I will now become very specific about how to go about establishing it. If you will indeed continue this work, the result will be a connection in many different ways—one will see, another will hear, a third will feel, and so on, in many different forms. The modalities in which the contact with God’s spirit world can become a reality are manifold.
因为我的教导的目标在于使与上帝的精神世界建立联络对每个人都成为可能，现在我会非常具体地说明如何着手建立它。如果你真的能持续进行这份工作，它的结果会是产生在一种许多层次上的联系——一个人会看到，另一个会听到，第三个会感觉到，等等等等。这种与上帝的精神世界的联络变成现实的方式是多方面的。

Some people, as you know, receive this special force before entering this path. This is what you call mediumship. When a human being is blessed with this capacity—and I hesitate to use the word blessed—it can be a great test. When the spiritual force is given to a person who does not know what its purpose is, for that person this gift will be a burden and a trial. The spiritual world intends this spiritual force to be a signpost so that the individual should find the reality of God and His spirit world more easily. This of course means finding the path of perfection. Unfortunately, this is often not understood. The more these forces work and manifest, the more unhappy will the medium be. The more stubbornly such a person insists on his or her old way of thinking, the lower the spirits will be who make use of him. This is, of course, a lawful consequence of the medium’s attitude.
有的人，如你所知，在进入这条道路前就收到了这种特别力量。这就是你们称为媒介传导的东西。当一个人得到这种能力而受到祝福时——这里我对使用“祝福”这个词有犹豫——它可能是一个重大考验。当这种精神力量被给予一个人，而他不知道其中的目的时，这份礼物对这个人来说会是一个负担和测验。精神世界意在使这种精神力量成为一个指示牌，而让人更轻易地找到上帝和祂的精神世界这个现实。这当然也意味着找到自我完善的道路。不幸的是，人们通常不理解这一点。这些力量越多地产生作用并化为现实，传导人就越觉得不快乐。一个人越是固执地坚持自己老旧的思维模式，对他进行利用的魂灵的层次就越低。这种情况，当然了，是传导人持有态度的相应合法后果。

It is also possible that the person does know about “these things” and does cultivate his or her mediumship in certain respects—I should say merely outer aspects—but does not personally walk the path of perfection. Again, as in the first case, this kind of connection to the spirit world is a danger for the medium as well as for the people who surround him or her.
另一种可能是，这个人其实知道“这些事情”并且也在某些方面培养他或她的媒介传导能力——我会说仅在外部方面——但在他自己个人层面并没有走在自我完善的道路上。同样，如同第一个情况，这种与精神世界的连接对于传导人本身和他或她周围的人来说都是一种危险。

When, however, the force manifests after the person has entered the path of perfection, of development, of self-knowledge, of self-discipline, of humility, when he or she crosses the first threshold and uses the contact only for this one purpose, in that case mediumship will evolve in the most wonderful fashion. Then there will be none of the very specific, difficult trials, the details of which I will not describe now, that come when a medium believes she can help others without entering this path herself. I do want to make the point, though, that spirit contact must only serve one purpose:  spiritual development. When this truth is recognized, help will come. And only then can the medium help others and can others be helped by her. I will talk more about this later.
但是，当这种力量显现在一个已经进入自我完善、自我发展、自我认知、自我纪律和自我谦卑的道路上的人身上，当他或她越过了最初的临界点并且把这种联络单纯地用于这一个目的时，在这种情况下，传导能力会以最奇妙的方式产生进化。这时候就不会再有那些非常特定、困难的测验（也就是当一个传导人相信她可以在自己不进入这条道路的情况下帮助别人时会遇到的那些测验，我在这里不再描述）。但是我想提出：精神联络必须只用于完成一个目的：精神发展。当这个真理被认清后，需要的帮助就会到位。也只有这时候传导人才能帮助别人，别人才能得到她的帮助。我后面会继续谈到这。

When people are so far evolved that they receive answers to their questions relating to their own development, then their faith will be immeasurably strengthened. The faith that used to be shaky will now be solid.
当人们发展到足够远的时候，以至于他们能收到关于自我发展问题的答案时，他们的信念会无法衡量地增强。之前动摇的信念会变得坚定不移。

Now, if you asked me, “How can I start?  What should be my next step on this path?”  I would answer that everyone has to begin differently. A spiritual problem that needs to be cleared away first may otherwise obstruct the next stretch on the path. It may be a rigid prejudice that would make it impossible to be open to someone else’s words. One must identify the problem and ask to be open about it. However, no answer will be forthcoming unless you open yourself so that you can say, “I am ready to accept the answer if it is God’s truth, even when up to now I believed its opposite.”  Open-mindedness has to be always present on this path. A person has to devote much work, willpower, and time to reach this state of mind. If you have not invested all that, then you need to ask yourself, “What are my imperfections?  Where are my faults?  What is my greatest fault, which might be the greatest obstacle to my spiritual ascent?  What are the faults which I have already recognized, but about whose consequences and extent I have not given myself a clear account?  And what are the faults with which my fellow-humans have confronted me?”  Think about them. You will find yourself saying, “This is unjust, it is not true that I have these faults”—and you might even be right. Yet there must be a grain of truth in such opinions, perhaps differing from how they were expressed to you or how you understood them. Consider them with honesty and with absolute goodwill to get to the truth. The greater your resistance, the likelier it is, my dear ones, that you can find the grain of thrush in what you have been reproached with and until now not acknowledged.
现在，如果你问我：“我怎么开始呢？我在这条道路上的下一步怎么走？”我会回答说每个人的开始方式并不一样。某个特定的精神领域问题需要首先被清理掉，否则它就会阻碍道路上的下一步。它可能是一种僵硬的偏见，这会让一个人没法以开放的态度对待别人说的话。人必须看清这种问题然后真诚面对它。但是，没有什么答案会呼之即来，除非你敞开自己并让自己这么说：“我已准备好接受答案，只要这个答案是上帝的真理。哪怕到目前为止我都在相信它的反面情况。”人在这条道路上必须永远保持头脑开放状态。人必须倾注很多工作、意志力和时间才能达到这种思想状态。如果你还没有投入这一切需要的准备，那么你要问自己：“我的不完善之处有哪些？我的过失有哪些？我最大的过失是什么？那个阻止我在精神领域上升的最大障碍是什么？我已经认识到，但是对于其后果和程度还没有给我自己一个清楚说明的过失是什么？以及我的人类同伴们与我对峙时提到的过失是什么？”把它们思考一下。你会发现自己说：“这不公平呀，我不可能真有这些缺点”——而你甚至可能是对的。然而在这类意见中一定会有少许真理存在，它们也许会不同于传达给你时候的样子，或者不同于你一开始理解或解读的样子。以诚实的态度和绝对的好意来考虑它们，这样你才能靠近事实。你的阻力越大，越有可能的情况是，我亲爱的人们，你会最终发现那少许的问题确实所在，正是它使你受到之前的责备，直到现在才被你承认。

Make a list of these faults so as to keep them in your awareness and thus prevent them from disappearing again. This will help you very much. Then take the list to your personal spiritual friends—whether in the body or not—and by opening yourself, you will receive an answer, a deeper understanding, and will perceive connections. Whoever is ready to hear even what is unpleasant, saying, “Father, your will be done, I want the truth about myself and will not recoil from it in cowardice, vanity, and oversensitivity,” will indeed receive the truth.
把你找到的这些弱点列成一个单子，用这种方法你能一直对它们有所觉察，而不会再次疏忽掉它们。这会使你受益良多。然后拿着这份单子去找你个人的精神层面的朋友们——不管有形的还是无形的——然后通过敞开你自己，你会收到一个答案、一种更深的理解，然后你会发现其中的联系。不管是谁，只要他做好准备听取哪怕是令人不愉快的事实，说到：“天上的父，你的意志会得到完成，我想要关于我自己的真理，我不会因为懦弱、虚荣和过度敏感而退缩回来。”那么他就真的会得到相应的真理。

Your happiness will be greater when the truth coming through is one that you resisted, because it will be a greater confirmation of the reality of the experience than hearing an answer which you had actually hoped for. In the latter case, you could tell yourself that the answer was only wishful thinking. When, however, something that has been resisted comes through, then, my dear ones, you will actually feel the presence of your spirit friends, and that confirms the presence of that other world, the existence of which is so often doubted, the world which you cannot see and touch. Such an answer will steady your weak faith. Consider it as the first answer from God, given only when a human being is willing to take the first step, that of overcoming his or her resistance to change. The reward is great, because the security which comes from getting personal answers and knowing the reality of the spirit world through one’s own direct experience cannot be obtained from any amount of words, whether heard or read. But the “outer” words are also necessary; they give you the impetus to do what is needed.
当从中得到的真理是你之前对它产生过抵抗的那种真理时，你的快乐会更大，因为相比你实际想听到的回答来说，这种回答带来的对于你经历的真实性的证实要大很多。而如果是你得到了自己实际想听的答案的情况，你可能告诉自己说这个回答只是自己的一厢情愿的瞎想。然而，当某些之前一直被阻挡的东西穿透过来时，这时，我亲爱的人们，你会真切地感受到你的精神朋友们的存在，而这又会证实另一个世界的存在，也就是那个存在性总是受到怀疑、你看不见摸不着的世界。这样一个回答会使你的脆弱信念变得平稳。你可以把它看成来自上帝的第一个回答，这个回答只有在一个人愿意迈出第一步，也就是在克服他或她对于改变的阻力时才会给出。而由此获得的奖励是丰富的，因为从“收到个人答案”以及“通过自己的直接经历来了解到精神世界的现实性”这个过程中得到的安全感，是从再多的语言中——不管是听到还是读到——都没法得到的。但是那些“外部”的语言也是必要的；它们给你去做必要铺垫的促进力。

This should be the beginning of your path. After that you will notice little things every day and then can ask your spirit friends,  “Help me to recognize the meaning and purpose of this particular experience and see how it might further my development.”  Again you will receive answers. I wish to warn you, though, that the preparation for and eventually the hearing of something that is uncomfortable must take place again and again, not mechanically, but with an ever-renewed readiness. In no way does it suffice to do this only once. If the door is closed, truth cannot get through, and the door is closed when one is not completely ready to accept that which is uncomfortable and unflattering but true. If you recognize and follow this, then you will again and again receive answers through recognitions or sudden inner knowledge.
这应该是你道路的起点。在此之后你会每天注意到一些小事情，然后请求你精神界的朋友们：“请帮我认识这件特定经历的意义和目的，以及它如何推进我的发展。”你会再一次收到回答。然而我想要告诫你们，要做好应对听到某些让你不舒服的东西，以及最终这样做的心态准备，而且这个过程必须一次又一次地发生，但不是机械地，而是带有一种不断更新的准备就绪的状态。仅仅一次这样做在任何情况下都是不够的。如果门被关上了，真相就没法穿过来，而当一个人没有完全准备好接受那些令人不舒服，或者不讨好人却是真实的情况的时候，门就会被关上。如果你能意识到并遵从这一点，那么你就会一次又一次地通过认知或者突然的内在领悟接收到回答。

An answer may come to you, perhaps a few days later, through the words of a fellow human being who is inspired. This is one of the ways God’s world helps you. Therefore you need to keep your eyes and ears open for a message. This is a test, an opportunity to learn humility by being open to such help, and by listening to another person who has something to tell you. This is then the second step for you to undertake. When you pass this test, a new, wonderful experience of deep recognition will follow that again confirms the reality of the spirit world. Such an experience has the same validity as when the answer comes directly from within. You will feel evermore clearly in what a wonderfully organized way you are guided on your path.
一个你要的回答，可能在几天后通过一个受到灵感激发的人类同伴传递给你。这是上帝的世界对你进行帮助的几种方式之一。因此你需要保持眼睛和耳朵的敏锐来收到一条信息。这是一个测验，也是一个学习变得谦卑的机会：通过对这种帮助保持开放的状态，你会达到这个目的；而对一个对你有话要说的人保持开放状态，你也达到了这个目的。所以这是你需要进行的第二步。当你通过了这个测验，随后就会产生一种新的、奇妙的深层认识的经历，它会再一次地确认精神世界的真实性。这种经历与答案直接来自于内心的情况有着同样效力。你会越发清晰地感觉到，自己在道路上正在以多么奇妙的、有条不紊的方式受到引导。

I would like to add that whoever continues on this path and reaches a certain solidity in the work will one day come to the point of feeling the need to give God something back in gratitude. At first, it may seem as if a sacrifice were being asked of you—a sacrifice of time, of course, but in addition a commitment which feels burdensome. It feels like a sacrifice to let go of something, to overcome something with a great effort. You see this in the beginning only as a hardship. You focus on the price you must pay, on what you have to give. The happiness that will come from such victory over oneself seems in the beginning only a promise whose meaning is not understood. Yet whoever is ready to begin to pay the price will soon recognize that one gets a hundred times more than one gives.
我想补充的是，任何人只要在这个道路上坚持下去，并在其中达到一定可靠性的话，那么有一天他会到达一种境界，会产生一种出于感激而想对上帝回报的需要。一开始，看起来好像你被要求做出一种牺牲——当然了，是时间上的牺牲，而且还有一些承诺感、投入感，这会使人有负担的感觉。通过极大的努力去放弃或者克服一些事情，这感觉起来会像做出牺牲。你一开始会把它看成一种艰苦的条件。因为你把关注点放在了你必须付出的代价和你必须给出的东西上面。这种对自我的胜利带来的幸福快乐一开始显得仅是一种承诺，其含义未被理解。然而只要他开始做好付出代价的准备，就会很快发现：一个人得到的会是他付出的一百倍。

I know that these are only words for those who are not that far in their development. Remember my words, though, because you will find them proven. Then you will be ready to offer your service to God’s great plan, to become His co-worker. All you will need is readiness; the how must be left to God. You will be guided to find the best way you can help. Again, it will be evident to all of you who make this choice that the willingness to help and serve is not a sacrifice, but becomes the greatest joy imaginable.
我知道这些话对那些进展还不多的人来说仅是空洞的语言。但是记住我的话，因为你会发现它们最终将被证实。那时你就会准备好把自己提供给上帝的伟大计划，成为祂的同事。你需要做的只是做好准备；具体怎么做的问题必须交由上帝处理。你会被指引到最能发挥作用的地方。再一次地，你们这些做出了选择的人们会明确无误地发现，去帮助和去服务的意愿并不是一种牺牲，它会成为你可以想象的最大快乐。

I can see for two of my friends here very definite tasks in this connection. I see for you, my friend [a man who has been an investigator for psychic research, cooperating with scientific circles, and has had much experience with mediums] that you will have the opportunity on this path to train mediums. You will teach them to establish contact with God’s spirit world and to learn the requirements, conditions, and laws that regulate such contact. More and more people with mediumistic talents will be led to you. You will be able to help many, to raise their practice to a higher level and thus help in the Plan of Salvation. A wonderful task awaits you, but before it can be undertaken you need to fulfill some other conditions.
我可以看到有两位在座的朋友在这方面有很明确的任务。我看到你，我的朋友【一个灵学研究的调查者，与科学圈合作并对媒介传导很有经验的人】在这条道路上会有机会培训传导人。你会教导他们建立与上帝精神世界的联系，并帮助他们学习其中的需求、条件以及调控这种联络的法则。越来越多拥有传导天赋的人会被指引到你那里。你会有能力帮助很多人，把他们的实践水平提升到更高层次，以此为拯救计划做出帮助。一个奇妙的任务在等着你，但是在开始之前你需要满足某些其他条件。

I want to add that the good already created by some mediums in convincing certain circles of the reality of the spirit world will by no means be diminished by the contact with God’s world rather than by contact with spirits not yet enlisted in the Plan of Salvation. Exactly the kind of proof which skeptical people want to see can be also furnished by God’s world although in a different way, with other conditions, even though the inquirer has to learn that he or she cannot specify exactly how something should be proven. When a person is prepared, however, to wait patiently—and one can explain this to the people who want proof—then the proofs coming from God’s enlisted spirits are incomparably more effective and enduring than those furnished by the still blind spirits, which might produce strong momentary impressions but whose power quickly dissipates. People generally believe that skeptics cannot be convinced unless spirits who are like humans are at work and influence humans in their way. This, however, does not have to be so. Science could obtain more knowledge of this kind if the observers did not force their own conditions upon the spirits, but would accept the conditions for communication set by the spirit world. This is by no means so difficult. All that is needed is to enlighten people about the different conditions required by the spiritual laws.
我想补充的是，一些传导人在使某些圈子的人们相信精神世界的真实性的过程中已经创造出了一些正面效果，这种好的效果肯定不会因为与上帝世界的联络而受到贬低，它只会受到与那些还没有加入拯救计划的魂灵们的联络而受到一些影响。充满怀疑性的人们总是会要求某种证据或证明，这种类型的证明同样可以由上帝的世界提供，不过是以不同的方式、在不同的条件下，而且提问者必须学到一点就是，他或她无权指定某件事到底应该怎样被证明。但是，当一个人做好准备并且耐心等待时——你可以把这个原理解释给那些想要“证明”的人们——这时候来自与上帝同盟的魂灵们的“证明”就会比那些仍然处于盲目状态的魂灵们给出的“证明”不具有可比性地更为有效，因为后者也许能给出一些短暂瞬间内很强的影响力但是它们的力量会很快消散掉。人们普遍相信的是，怀疑论者没法真正被说服，除非是有思维方式像人类一样的魂灵们在产生作用，并且是以那些喜欢怀疑的人们认可的、能接受的方式影响他们。但是，事情并非一定要这样。科学认知系统其实可以获得更多这类知识，条件是观察者们不把他们的人为条件强制要求在魂灵们身上，而且科学家们需要接受由精神领域世界设立的对于这种沟通的需求条件。这根本不是什么困难的事。关键仅仅在于人们要明白精神法则所提出的不同条件而已。

In addition, I see yet another task for you, my friend. You will slowly begin to see people’s auras. Initially the aura will manifest occasionally, at first hazily and only partially. When you continue to cultivate the perception, your talent will develop further. You will not only see the colors of the aura but will distinguish patterns. The human aura displays varying patterns, some of which can be recognized by human beings, for instance illnesses, soul-attitudes, and unspoken thoughts. All this, of course, has to be learnt, and you will receive help to develop this ability.
另外，我看到了你的另一项任务，我的朋友。你会慢慢开始看到人们的光环。最初这些光环只会偶尔显现出来，一开始显得朦胧模糊，而且只是部分可见。当你继续培养这种感知能力时，你的天赋会进一步发展。你将不仅能看到光环的颜色，还能分辨出模式和图案。人类的光环会显示出不同的模式，其中有些能被人类自己识别出来，比如疾病、灵魂态度以及未说出口的想法。所有这些，当然了，必须通过学习得到，而你会得到帮助来培养这种能力。

You might ask what the purpose of such a talent may be. Remember that nothing is given without a purpose. Sometime later in the future you will be given healing powers. When you perceive the aura of a person, you will perhaps see patterns of an illness within the aura and thus know what the illness is. You will receive the corresponding power to heal it. This wonderfully fulfilling task is still awaiting you. God directs it in such a marvelous way that it will be the greatest joy for you without interfering with your regular duties. Have trust and let yourself be guided. In fact, offer it up to God.
你可能会问这样一个天赋的作用是什么。记住，没有什么事是漫无缘由地被给予的。未来一段时间以后你会被给予治疗方面的能力。当你感知到一个人的光环的时候，你可能会看到光环中存在某种疾病特有的模式，进而知道这个人的疾病是什么。你会得到相应的治愈它的能力。这个极为有意义的任务正在等着你。上帝以如此不可思议的方式引导着它，以至于这会成为你最大的喜悦，而且它不妨碍你的日常职责。让自己具备信任，让自己接受引导。事实上，你可以把它交给上帝。

I ask all of you to adopt the same attitude toward all aspects of the work. Ask, “When should I let myself be guided?  When must I act by myself?”  These questions can also be asked about your personal connection with the spirit world.
我请求你们所有人都以同样的态度面对这份工作的所有方面。可以这样问：“我在何时应该让自己受到指引？哪些时候我应该自己行动起来？”你也可以就你个人与精神世界的关系提出相同的问题。

I have received these two messages to deliver to you. Other messages will follow, coming either through me or through a personal contact. Now those of you for whom it was not the time to receive similar messages should not think, “Why not me?”  For everything has its good reasons. Trust—even if you cannot always understand the reason. This is how all of you should try to establish contact with the spirit world. It is available for every single human being. Everyone. Do you think that God makes contact possible only for specific people?  No. This is not so.
我收到了以上两条消息来传递给你。其他的消息会接着来到，要么是通过我，要么是通过某个个人联络方式。好了，你们其中有些人还不到接收类似消息的时候，因此不要觉得“为什么不是我？”因为一切都有它各自的道理。选择信任——哪怕你不总是能够理解其中的原因。这是你们所有人应该具有的对于建立与精神世界联络的心态。这种联络对每个人类个体都是可及的。所有人。你认为上帝会把这种联络的可能性限制于仅仅某些特定的人吗？不。不是这样的。

All conscious and unconscious strivings of humankind aim at penetrating the heaviness of matter. What, after all, is the meaning of your technological progress?  Think about it. Is it not to conquer matter?  Each technological achievement is nothing but an overcoming of matter which is now an obstacle to you. Technology, however, achieves this with outer forces, while the person on the path of perfection utilizes inner forces. That is the entire difference. The latter method is incomparably more effective.
人类所有的清醒意识与无意识的努力奋斗，目标都在穿透物质本身具有的沉重属性。归根结底，你们的技术进步的意义究竟是什么呢？考虑下这一点。它难道不是为了征服物质世界？每一个技术上的成就不过是在克服某个现今对你们来说是障碍物的物质。然而，科学技术是通过外部力量达成这一点，而走在这条道路上的人运用的是内在力量。这就是整个区别所在。后者在方法上具有无可比拟的有效性。

And now I will try to answer your questions to the best of my ability.
现在我会竭尽所能回答你们的问题。

QUESTION:  I would like to know why, in our times, it is so difficult to establish connection with the spirit world.
问题：我想知道为什么，在我们的时代，想要建立与精神世界的连接是如此的困难。

ANSWER:  In general, the connection to the spirit world is not made difficult. It is humans who hinder it through their overall orientation. Most people do not believe in these things and are not open to them. The mediumistic powers are dormant in many people and, instead of being put to use, they lie barren.
回答：总体来说，与精神世界的连接并没有被设立得很困难。是你们人类因为自己的整体定位取向对这个过程进行了阻碍。多数人不相信这些事情，并且没有对它们保持开放的态度。作为媒介进行传导的能力在许多人中保持休眠的状态，这种能力没有被加以利用，因此毫无作用地潜伏着。

But perhaps what you really mean is not a general connection to the beyond but especially to God’s spirit world. This kind of connection was always difficult to establish because, obviously, it is the most precious gift that a human being can ever obtain. It cannot be handed over to humans easily and simply, without any effort on their part. God’s spirit world must test a medium again and again, to see whether he or she is worthy and capable. The medium must not only be morally unobjectionable, but his or her attitude must be the best, befitting a free and unconstrained soul. Many good people fulfill the first requirement and have the wish and will to help others, even to the extent of entering the path of perfection. But many do not persist on this path and get caught in a fault they fail to recognize and therefore fail to conquer. For a medium the great temptation after having reached a certain level of success is to succumb to pride instead of mastering it. Complete humility is a requirement. Consider how difficult it must be for a medium not to identify with the manifesting spirit and thus not to elevate her- or himself above others. Relatively many are successful up to a certain point and then get stuck.
然而也许你实际意思指的不是与所谓“上面”的联系而是特指上帝的精神世界。这种类型的联系从来都不是那么容易建立的，因为显而易见的是，这是一个人所能获得的最为珍贵的礼物。它不可能被轻易地、简单地交给人类，而不需要他们付出自己分内的努力。上帝的精神世界必须一次又一次对传导人进行测试，来判断他或她是否是符合信任并且有能力。传导人不仅必须在道德上无法辩驳，并且他或她的态度必须是最佳的，因而能够适合一个自由的、不受拘束的灵魂。许多善良的人满足了第一个条件，也具有帮助别人的愿望和意志，甚至达到了进入自我完善道路的程度。但是很多这样的人没有在这条道路上坚持不懈，然后陷入了一个他们没能够识别出来的弱点之中，因而在征服这个弱点的过程中（暂时）失败了。对一个传导人来说最大的诱惑在于，在达到了一定程度的成功后，会很容易屈服于自己的骄傲，而不是掌控它。完整彻底的自我谦卑是要求之一。考虑一下，对于一个传导人来说想要不把自己与通过自己显现的魂灵混为一谈，并且不因此把自己放在高于别人的位置上，是一件多么不容易的事。相对多数的人在一个特定的临界点之前都是成功的，然后就卡住了。

As I have already explained in this lecture, no one can help others unless most actively pursuing this path of perfection and being solidly rooted in it. You should therefore test all mediums from this point of view. Most of them will tell you that they pray and strive to be decent people. However, they do not develop themselves steadily in the sense necessary to heal the sick soul currents present more or less in all people. For each unrecognized, unconscious emotion, each unrecognized fault that a person does not work on, each prejudice is an impenetrable obstacle for God’s spirits. It is not that they are forbidden from manifesting because the medium has not deserved it. It is that they are incapable of doing it. What people do not recognize in themselves, what deviates from spiritual law, cannot be penetrated by us spirits. Each rigid opinion is such a block.
如我已在本次讲座中解释的，没人能够帮助其他人，除非他（她）非常积极地追寻这条自我完善的道路，并且坚实地扎根其中。因此你需要以这种观点检验所有传导人。他们绝大多数会告诉你说，他们经常祈祷并且为成为体面的人而奋斗。但是，他们并没有在“治愈或多或少地存在于所有人内心的病态灵魂暗流”这个意义上去持续地发展自己。因为每一种没被识别清楚的、无意识的情绪，和每一份没有被识别清楚的弱点，只要这个人没有对其加以应对，那么（他/她的）每个偏见对上帝的魂灵们来说都是一个没法穿透的障碍物。不是说它们因为传导人还不值得它们这样做，而被禁止显现自己。而是说它们能力上做不到这一点。人们在自我中没有识别清楚的东西，以及偏离精神法则的东西，是被我们魂灵们无法穿透的。每个僵硬的意见主张都是这样一个障碍物。

We cannot push the truth through when the person is not open to the possibility that the truth may be the very opposite of his or her tight conviction. I am not referring to the basic requirement that a medium must commit his or her talent only to God’s will and not claim any personal advantages, either material or spiritual. Even a person who fulfills the requirements as well as possible may yet deviate slightly from spiritual law and will therefore present an obstacle to us. Thus you find hardly ever a complete and perfect instrument. It is difficult to make you understand this because the realm of spirit, its movements, emotions, and thoughts are for you like air, but for us they are solid forms, walls built of granite.
我们没法把真相强推出去，特别是当事实可能与他或她自身紧绷的信念相左，而这个人没法接受这种可能性的时候。我指的不是对于传导人的基本需求，也就是说他或她的天赋必须单独用于上帝的意志而不以此获取个人的好处，不管是物质的或是精神的。就算一个人尽可能好地满足了要求的时候，也有可能时不时地略微偏离精神法则，这会为我们呈现出障碍。于是你很难发现一个完整又完善的工具。这不太容易给你们解释清楚，因为精神的领域，它的运动、情绪和思想对你们来说只是空气，但是对我们来说却是实在的形状，如花岗岩构成的围墙。

When, however, a medium fulfills most of the requirements, when he or she develops steadily and does not think of having reached the goal once and for all—because that does not exist—then those little obstacles do not matter. Then one can go around them, if I may say so. Also, an obstacle can serve as a valuable test which will only benefit the person, for every test is beneficial when successfully passed. After reaching a certain level, small obstacles will not stop either the medium or the audience from reaching the truth, in spite of the still existing imperfections in the medium. To help reach personal perfection through overcoming the obstacles is, after all, the only objective of God’s spirit world, the only reason for the connection.
但是，当一个传导人满足绝大多数需求，当他或她稳定持续地发展自己而不是想着要一劳永逸地达到目标——因为这是不存在的——的时候，这时那些小障碍就无所谓了。这时人就可以绕过它们（如果我可以这样说的话）。而且，一个障碍物对一个人来说可能是一个宝贵的测验，它只会使这个人受益，因为每个测验在被成功通过的情况下都是有益的。在达到一定的层次后，小的障碍物就不会对传导人和听众到达事实和真相而构成障碍了，尽管传导人本身还存在不完美之处。毕竟，帮助人们通过克服障碍来接近个人的完善状态，是上帝的精神世界的唯一目标，也是建立这种联系的唯一原因。

QUESTION:  Various religious philosophies differentiate strongly between dualism and monism. Is dualism a particular level leading to monism, and is monism the higher truth?
问题：不同的宗教哲学观点在二元性和一元性之间存在强烈分歧。二元性是通向一元性的一个层面吗，一元性是更高的真理吗？

ANSWER:  One can look at dualism as a level. To explain this is an endlessly difficult undertaking and for human beings to grasp it is difficult. But the present difficulty is that in this respect I cannot manifest through the medium. This is an example of the limitations of mediumship. She is open enough so that I can get these words through her. I could not say this through many mediums, but in her case the openness is there, at least to this extent. Her development, however, has not progressed far enough for me to get the full explanation across. Perhaps in a few months, or a few years, perhaps even in less time, this will be possible. I do not know now. I will answer the question whenever her development is far enough advanced to eliminate the block.
回答：二元性可以被看为一个层次。要解释清楚这个问题是个无比困难的任务，而要人类掌握它是很难的。但是眼前的难度在于在这一点上我没法通过这个传导人进行显现。这就是一个关于传导媒介的局限性的例子。她有足够的开放程度，因此我能通过她说出这些话。我通过很多传导人是没法这样说的，但是在她的情况里开放的态度是存在的，至少是在这个程度上。但是，她的发展程度还没有到能使我能把完整的解释表达出来的程度。也许是在几个月后，也许是几年，甚至可能在更短的时间内，这会变为可能。我现在不知道。当她的发展达到了足够消除其中的障碍时我会回答这个问题。

QUESTION:  Does this mean that one is quite dependent on the state of the medium?
问题：这是不是意味着一个魂灵会很依赖传导人的状态？

ANSWER:  No. When people have reached the stage of development that a connection with God’s spirit world can be established, the spirit world will make sure that the people who use this contact will always be given what food they need, in spite of their imperfections. The communication will always promote their development and bring them closer to God and the state of bliss. As I have already mentioned, it is possible for us spirits to bypass the blocks that are in the way. Therefore everybody can receive what they need to attain the highest level they are capable of. That is really all you need, since the goal of the contact cannot be anything else. You are able to distinguish between the spirits who belong to the divine order and can help and teach and other types who cannot. Everything else is mere detail and unimportant.
回答：不。当人们到达了某个发展阶段从而能够建立起与上帝精神世界的联系时，精神世界会确保那些利用这种联络的人们总能得到他们需要的食物，尽管他们存在不完美之处也没关系。这种沟通总会增进他们的发展并且让他们更接近上帝和极乐的状态。如我之前提到，对我们魂灵们来说绕过挡路的障碍物是可能的。所以每个人都可以收到他们需要的东西，去达到力所能及的最高层次。这真的是所有你会需要的东西，因为联络的目的不可能是其他的。你有能力区别出哪些魂灵是来自于神圣秩序并可以帮助和教导人们，哪些魂灵无法这么做。其余的只是细节而且不怎么重要。

A spirit of God will always insist that every person should establish an independent connection to God’s world as soon as such a connection becomes possible. The spirit will help and give directions. Of course, at the very beginning of a medium’s work, when there is still a need for tests, some untruths will enter occasionally because of the still existing blocks. But this is not important, my dear friends. Everyone who is wholly determined to develop will always look inside and meditate sincerely whether a personal message is exactly right or not and whether he or she can learn something from it. The most important factor is to be fully aware of the basic conditions, laid down in all religious philosophies:  to follow the will of God, to purify oneself by overcoming one’s faults, and to cultivate self-knowledge.
一个来自上帝的魂灵会一直坚持说每个人都应该建立起独立的与上帝世界的联系，这个联系越早越好。魂灵会提供帮助并给出指导。当然了，在一个传导人工作的最开始阶段，当仍然有需要进行测验的时候，有些非真理偶尔会进入，这是因为有障碍物仍然存在。但是这不重要，我亲爱的朋友们。所有完全坚定于自我发展的人总会进行真诚的内省和冥想，来弄清某个个性化信息到底是对是错或者他或她能否从中学到什么。最重要的因素是要全面意识到基本的要求，同样的基本要求铺设于所有的宗教哲学之中：去追寻上帝的意志，去通过克服弱点净化自我，并培育自我认知。

As to the greatest existential questions of Creation, you will have the right attitude if you understand that to know the answers fully is not so important. Wait with humility until the knowledge becomes accessible, until you have progressed enough to deserve it. Simply follow the Bible:  “Test everything and keep the good.”  Quietly accept what serves God’s will. Accept the message with a humble and patiently detached attitude, without wanting to know everything at once.
至于那个世间最大的存在主义问题，如果你能够理解“掌握所有的答案并不是最重要的”这个事实，那你就会拥有正确的态度。以谦卑的态度进行等待，直到相关的知识变得对你可及，直到你取得了足够进展而使自己变得值得得到它。只要简单地听从《圣经》所言：“尝试一切然后留下好的。”安静地接受那些为上帝的意志服务的东西。以一种谦卑和耐心超然的态度接受这信息，而不要想着一次性地知道所有事。

The right attitude also includes not being dependent on the medium and never on the spirits. When this is the case you cannot be damaged, even when a medium is still in a state of training and transition. Do not put all your eggs into one basket. What matters is clear thinking and humility. After all, is it not a lack of humility to assume that you human beings have already deserved to be given the answers to the greatest questions, to assume that you can have direct contact with God’s world without first having proven yourselves?
正确的态度也包括不要依赖于传导人，也绝不要依赖被传导的魂灵们。如果你能做到这一点，那么哪怕传导人仍然处于培训和过渡状态，你都不会受到伤害。不要把你所有的鸡蛋放在一个篮子里。关键在于清晰的思维和谦卑的心态。毕竟，你们以为人类已经值得得到最伟大的问题的答案，以为你们可以不必首先证明自己就能够拥有与上帝世界的直接沟通，这难道不是缺乏谦卑的表现？

What I say applies not only to mediums but to all of you. My warning that one should not accept the medium’s messages unquestioningly can be easily misused by those who don’t want to look at a sensitive area hidden from consciousness within themselves. To accept or dismiss disquieting words can also be a test. When you are not able to find an inner echo to a message which causes you discomfort, be very careful before rejecting it. Ask God again and again for the inner truth. You may then recognize the very words you have tried to dismiss to be indeed the truth. Such an insight will produce great inner joy. Whenever you have a personal question, you need to approach it with this attitude. With a general question, however, it is wise to adopt a patiently detached and humble attitude, because all knowledge in this area is a gift of grace which has to be earned by everyone. When the medium and the participants successfully pass the tests—and everyone can who is willing—then, in time, more and more will be given. As far as the large general questions are concerned, the complete answer is often way beyond the capacity of human understanding. You have altogether too many false opinions—not only you who are here but humanity in general—even about things you can easily explore in your own world, without the help of a spirit. Whether you know less or more about the great questions relating to Creation, whether you have more or fewer false ideas is not the issue, since you have already so many misconceptions in purely earthly matters. The issue is rather to develop spiritually and to accept spirit advice when you know without doubt that it is right. If you are disposed to do this, you will inevitably receive more and more confirmation that you are on the right road. This will strengthen your trust and your connectedness to God’s world—be it personal or through a medium.
我说的话不仅适用于传导人，而且适用于你们所有人。我做出的关于一个人不应该毫不质疑地接受传导人的话的警告很容易被那些不想审视自己隐藏于无意识中敏感区域的人所误用或滥用。对于使你不安的话是接受还是驳回，这本身也会是一个测验。当你遇到一个使你不安的消息但在内心没有相应的对应时，你在拒绝它之前需要非常小心仔细地加以辨别。一次又一次地请求上帝给你所需的内在真理。你可能会发现那些你试图驳回的话其实确实是事实。这种洞察会使你内心产生很大快乐。每当你有一个个人问题，你都需要以这种态度处理它。但面对一个一般性的问题时，明智的做法是先采取一种耐心超然和谦逊的态度。因为这个领域的所有知识都是一种恩典，它必须被每个人通过努力赢得。当传导人和参与者成功地经过考验——而且每个有这种意愿的人都可以——那么最终，会有越来越多的知识理解被给予你们。至于那些普遍性的大问题，对于它们的完整回答通常情况下远远超出人类理解力的水平。你们有太多的错误见解——不仅仅针对你们在座的而是对人类社会总体而言——甚至是对于不需要魂灵帮忙的、在你们世界中可以轻易探索的事情，也是这样。你们对于宇宙造物的重大问题知道的多与少、你们的错误观点是多是少不是问题关键，因为你们对纯粹世俗的问题都有如此多的误解。关键在于进行精神发展，并且当你确认无误时你能接受精神方面的建议。如果你倾向于这么做，那你就会不可避免地收到越来越多的证实，表明你正处在正确道路上。这会加强你的信任和对上帝世界的连接，不管是个人层面的还是通过传导人得到的。

I want to make it quite clear to you that this particular contact is well grounded, which becomes evident from the fact that this message is coming through. Obviously, the medium is sufficiently open and free of vanity. When you test several mediums—and they should be tested in this particular respect—you will be convinced that very rarely does one come across a medium who is free to such a degree. Precisely this evaluation should serve as proof of the reliability of this connection and of her good progress in the right direction.
我想说清楚的是现在这个联络是非常扎实的，这一点从“这份消息能够传递出来”这个事实中表现得很明显。显而易见的是，这位传导人有足够开放的心态并且不受虚荣心困扰。当你对几个传导人进行测验时——他们确实应该在这方面受到测验——你会很确信地发现很少能碰到如此不受拘束的传导人。这样的评价应该作为这个联络的可靠性和她在正确方向上取得了良好进度的证明。

Returning to the question about monism and dualism that our friend has asked, the only thing I can say is this:  Both monism and dualism exist. God is person and is also the One. I cannot bring through more now.
回到那位朋友提出的关于一元性和二元性的问题，我唯一能说的事情是：一元性和二元性两者都存在。上帝即是人也是一。我没法在此表达出更多细节。

QUESTION:  What I want to ask is difficult and perhaps stupid.
问题：我想问的问题有些困难而且也许有些愚蠢。

ANSWER:  Try it.
回答：试试看。

QUESTION:  I struggle with this problem again and again. If God has a Plan of Salvation for us, and if we have fallen away from God because we entered the wrong path, not the divine, why did God make His Plan of Salvation so complicated and so terribly difficult for us?  I know that it is necessary for our development, but it still seems too hard.
问题：我一次又一次地在这个问题上与自己内心斗争。如果上帝有一个针对我们的拯救计划，如果我们因为进入了错误的、非神圣的道路而已经离开了上帝，那么为什么上帝把祂的拯救计划做得对我们来说这么复杂，而且如此艰难？我知道这对我们的发展是有必要的，但它还是显得太难了。

ANSWER:  It only seems so to you. The difficulties that come up are there because the individual beings could not have brought it about differently. My dear friend, before the earth plane was created in its present form, other ways were tried, but they were not successful. And therefore God created the present conditions. Unfortunately, the development can happen only through the path of suffering. Unfortunately, indeed. There was an attempt to arrange it differently.
回答：它只是看起来显得这样。出现这样那样的问题仅仅是因为个体的人们没法以别的方式使它们出现。我亲爱的朋友，在地球位面以现在的形态被创造出来以前，其他的方式已经被尝试过，但它们都不很成功。于是上帝创造了目前的环境。不幸的是，发展只能通过苦难的道路达成。这确实是不幸的。曾经有尝试以不同方式安排它。

For you personally I want to add the following:  Try to apply the words which I gave at the beginning of today’s lecture. When you create a contact with God’s world in the way suggested, you will be strengthened. You will understand what these words mean:  the grace of faith. That can be given to you when you fulfill the conditions. Study the words and you will get an answer.
对你个人我想补充如下内容：试着应用我在今天演讲一开始给出的话。当你以推荐的方式建立与上帝世界的联络时，你会被增强。你会理解这些词的意思：信念的恩典。当你满足条件后这份恩典会被给予你。学习这些话你就会得到一个回答。

What goes on with you is the same that goes on with so many others. People say:  “God must show me His justice, then I will be ready to recognize Him.”  My dear ones, one day, when you have achieved the knowledge, you will come to understand how foolish such an attitude is, how mistaken both in thought and feeling. It is much better to start with the preconception that human beings are imperfect and not God and His Creation. You believe otherwise. Even when you do not think it through clearly, your emotions contain this attitude. I do not reproach you for it. So many people simply think that way. The same spite that originally led to the Fall is still present. It is an error in healthy, logical thinking. A revision of thinking is necessary. Look into the manifestations of Creation that you can see and to a certain extent even understand. Observe how wonderfully everything is fitted together, every smallest detail that is visible to you. Then consider how you demand that God should prove to you His greatest love and justice before you can allow Him to enter your heart. Can you understand that?
发生在你这里的情况也发生在如此之多的其他人身上。人们说：“上帝必须向我显示出祂的正义，然后我就会做好准备承认祂。”我亲爱的人们，会有一天，当你们得到了相应的知识，你们会理解到这种态度是多么的愚蠢，这种想法和感觉是多么的错误。不完美的是人类，而不是上帝和祂的宇宙造物，如果你们能以这种偏见看问题就会好很多。然而你们却相信是另一回事。哪怕当你们没有把它清楚地想明白时，你们的情绪中包含着这种态度。我不会因此责备你们。有这么多人就是这么思考的。最初引发堕落的相同怨恨在今天仍然存在。它是健康、逻辑思考中的一个错误。这里需要一个思考过程的修正。仔细观察你所见的并且在一定程度上能理解的宇宙造物，研究它的表现。观察下一切是以多么美妙的方式组成整体的，哪怕是那些你能见的最微小的细节。然后考虑下你是怎样要求上帝应该把祂最大的爱和正义证明给你，之后你才会允许祂进入你的心。你能理解吗？

QUESTION:  Yes, but often it is so difficult to see the will of God. I would very much like to follow it, but I don’t know which way to go.
问题：是的，但是通常要看到上帝的意志是非常困难的。我会很想追随它，但是我不知道应该怎么做。

ANSWER:  This is one example of what you should attain through the contact with the spirit world. Whoever wants to use this contact for the purpose of getting to know the will of God will certainly get a response. To him who knocks, the door will be opened. Whoever is determined to do the will of God will get the answer. But you always keep a little back door open as an escape route, a reservation; you are ready but at the same time you are also anxious that God might demand something unpleasant from you. This is going only halfway toward Him, exactly what I talked about in our last session. To go all the way to Him means, “Your will be done, God, even if it is unpleasant for me, even if I cannot yet comprehend it, because you are Justice and Love. That is clear, that is the only ground I can stand on, my only fixed point; I do not know any other.”  Of course, such thoughts have to be nourished so that they penetrate the feelings; they have to be formulated again and again; thought forms have to be built and rebuilt. With time, they will affect the feelings. Whoever has this attitude and asks for the will of God, without reservations and without keeping an open back door, will get the answer every time. Do you understand?
回答：这就是关于你应该从与精神世界的联络中获得什么的例子。不管是谁如果想通过这种联络来了解上帝的意志，他一定会得到一个答复。对敲门的人，门会被打开。任何坚定地想要完成上帝意志的人都会得到回答。但是你们总是保留一扇小后门作为逃跑路线，作为保留地；你们准备好了但同时你们担心上帝会对你们要求某些令人不愉快的事情。这只是朝祂走到一半，正是我上个演讲中讲到的。朝祂走到底意味着：“你的意志将得到完成，上帝，哪怕它对我来说不合意，哪怕我还没法理解它。因为你即是 正义 与 爱。这一点是清楚的，这是我唯一能落脚的地方，我唯一的固定点；我不知道任何其他的。”当然，这些想法必须得到培育从而使它们能穿透感觉；它们必须一次又一次地被构想出；思维形状必须被构建然后再重建。随着时间推移，它们会影响感觉。不论是谁以这种态度询问上帝的意志，不设保留余地并且不留后门，就能每次都得到答案。你理解吗？

ANSWER:  Yes, I understand.
回答：是的，我理解。

QUESTION:  How about the connection between the material and the spiritual, God and Mammon?  What is the right integration of the two?
问题：那么物质的与精神的，上帝与财富之间的联系是什么？怎样正确整合两者？

ANSWER:  Again, with money it is as with all other things. The principle is the same. When money is used as a means to an end, not a goal in itself, then the integration is right. When a person consciously determines not to be bound to matter by money but to use it like all the other gifts of God, such as health or any talent, in a spirit of gratefulness, to use the freedom from worries that money can provide to intensify his or her spiritual development, then the integration is just right. This is very difficult for many people. It seems easier to take the way to God when they are in a crisis, while the freedom from worries would make them deviate from the spiritual path. This, however, does not mean that it is necessary for the person who has material goods to give them away so as not to be burdened by possessions. You can enjoy wealth when you conquer the temptation that money, more than anything else, brings. It would be fanatical to say that a person has to give everything away and live in poverty so as to ascend spiritually. Such a task may sometimes be demanded from a person, but this is an exception. For instance, that person might be required to set an example to others, to those who say that it is no great feat to develop spiritually if one does not have material worries, but when one worries about the daily bread it is impossible to occupy oneself with such questions. It also happens that a very developed spirit is entrusted with this special mission during his or her earth life. If that is so, he or she will be led to it in meditation. But more often than not a person whose life is marked with an absence of material cares is tested to prove that he or she will follow the way of God in spite of the temptation, and is not captured by the spirit that rules the world of matter.
回答：同样，对于钱就像对于其他一切东西。原则是一样的。当钱被用作完成目的的手段，而不是本身作为目的时，这整合是正确的。当人明确地决定不被钱困在物质中而是像所有其他上帝的馈赠——比如健康或者任何天赋——一样，并且以感激的态度使用它，用金钱提供的自由和免于担心来增强他或她的精神发展，那么这个整合就是正确的。这对许多人是很难的。当人们处于危机中时转向上帝似乎是更容易的，而免于琐碎担心带来的自由使他们从精神轨道上偏离。但是，这不是说拥有物质财产的人们应该把它们送走而避免财产带来的负担。当你征服了金钱的诱惑时你就可以享受财富，金钱的诱惑在于比起其他任何东西来它更能带来你想要的。如果说一个人必须捐出一切，并活在穷困中才能获得精神升格，那就是狂热入迷了。这种任务有时会被要求在某个人身上，但这是特殊情况。比如说，这个人需要为别人做出榜样，别人指的是那些说“当一个人没有物质上的忧虑时要进行精神发展不算什么了不起的事，但是当一个人为了每天的面包而担心的时候要考虑这些问题是不可能的”的人。有时也会有一个非常完善的魂灵在他或她的世俗生命中被委托承担这份特殊使命。如果是这个情况，他或她会在冥想中被指引到这任务上。但通常来说，一个生活中显著缺乏物质关怀的人正在受到考验，来证明他或她在诱惑下会追随上帝的道路，而没有被掌管物质世界的那个魂灵捉住。

One cannot set up a generally valid rule for establishing the right middle ground; everything depends on the character. If a person leans toward greed, the right integration needs to proceed differently from another type who spends carelessly. Everyone must find his or her own middle ground, and you will certainly find yours on this path.
人没法建立一种普适的、牢靠的规则来找到中间位置不偏不倚的解法；一切都取决于个人品质。如果一个人倾向于贪婪，正确的整合就需要他与另一种挥霍无度的人以不同的方式进行。每个人必须发现他或她自己的中间位置，而你在这条道路上一定会发现属于你的（最适位置）。

QUESTION:  In other words, for those who strive spiritually, poverty is not a necessary condition, although this is assumed in many philosophies.
问题：换句话说，对那些进行精神奋斗的人，穷困不是必要条件，尽管很多哲学派别假定这一点。

ANSWER:  In no way does it have to be so, although it is possible in individual cases. When different philosophies establish this as a general principle, they do so because, again and again, it turns out that people fail to handle money rightly and find the path easier when they are poor. What is left out of consideration here is that all entities must, in the cycle of their incarnations, prove themselves capable of standing fast under any circumstances.
回答：绝不一定要这样的，尽管在个例中是可能的。不同的哲学派别把这设为通用原则，他们这样做是因为，一次又一次地，事实证明人们没能正确处理金钱，而且发现当他们贫穷时这条道路显得简单些。而这里没考虑到的原则是所有个体必须，在他们的降世循环中，证明他们有能力在任何条件下站稳脚跟。

QUESTION:  I perceive a great conflict in what the Bible says, which is that God created man in his own image and also gave him dominion over the earth. If I have been given dominion over the earth, I must also possess dominion over myself, my health, and everything. Consequently, I am capable to begin with myself first and then obtain rulership over the earth. I should be capable, since I am created in the image of God, to create whatever I want. But then it is no longer God’s will!
问题：我在《圣经》的说法中觉察到很大的冲突，那就是：上帝以他的形象创造了人，并且让他们统治地球。如果我被授予拥有地球，我一定也能掌握我自己、我的健康、以及一切。那么，我可以从自己开始然后获得地球的统治权。因为我是以上帝的形象创造的，那么我应该能够创造我想要的一切。但那就不是上帝的意志了！

ANSWER:  On this path you will come to know the will of God and you will come to will the same.、
回答：在这条道路上你会了解上帝的意志，而你进而会行使与上帝相同的意志。

QUESTION:  Through the attempt to establish the connection to God?
问题：通过与上帝建立连接的尝试？

ANSWER:  Yes, on the path of perfection you must develop the connection to God’s world. It will become ever more solid; your recognitions will increase steadily. You will come to know God’s will not only through what a spirit might tell you, but also through your own insights, through your own growth. Your spirit will become increasingly lighter, stronger, more aware. Your spiritual core develops, breaks open the husks, the crusts that surround your inner perfection and thus knows what the will of God is, not wanting anything else.
回答：是的，在自我完善的道路上你必须建立与上帝世界的连接。它会变得愈加稳固；你的认知会稳定增加。你会逐渐了解上帝的意志，不仅通过魂灵告诉你的东西，而且通过你自己的见解，通过你自己的成长。你的精神会变得愈加轻盈明亮、强壮、更富知觉。你的精神核心不断发展，打破外壳（也就是那些包围了你内在完美状态的外壳），因此你会知道上帝的意志，而不想要其他任何事。

QUESTION:  But this is a long way.
问题：但这条路很长。

ANSWER:  It is the same path I speak about and which all of you can follow. It means only happiness, ever greater happiness, ever greater liberation of all bonds, be they earthly, psychic or spiritual.
回答：这就是我说的道路，而且你们所有人都能追寻它。它只意味着幸福快乐、越来越大的快乐，越来越摆脱一切枷锁，不管是世俗的、心理的还是精神的。

My dear ones, I have given you enough for today. Think about what I have said. Go on in peace on this path. May the blessing of God flow through you, shine through you, and enable you to progress farther and farther and attain more and more happiness and joy. The blessing reaches to all my friends, near and far. Be with God.
我亲爱的人们，我今天已经给了你们足够多的内容。思考下我说的。平静地继续走在这条道路上。愿上帝的祝福通过你们流淌、发光，使你越来越多地进步并获得越来越多的快乐和喜悦。这份祝福会到达我所有的朋友们，不管远与近。与上帝同在。
